### PR TITLE
Improved the example code for `createCamera()`!

### PR DESCRIPTION
**Pull request by Brahvim**
Improved the example code for `createCamera()`! There is now a 'ground box' to help any new-to-OpenGL person viewing the scene understand what the camera is doing - moving on the `x`-axis while also rotating on the `y`-axis to look directly at `[0, 0, 0]`.

### CODE REVIEW

**Comment by davepagurek** [comment]
Thanks for adding comments to the example! I tested it out and noticed that the ground box is leaving trails behind:
image

This was probably happening earlier too but it was hidden because the small cube never moves by so much each frame to make the white part visible from behind its black outlines.

Would you mind also moving the `background(0)` call from `setup` to `draw`? I think that's all we need in order to fix that.

**Comment by Brahvim** [comment]
Whoops! Sorry, forgot to commit that in the pull request.
Added it in [this commit](https://github.com/processing/p5.js/pull/5873/commits/6ae654a379a03f0e26bbe864f34719e7d1ba68c9)!

**Review by davepagurek — APPROVED** [review]
Awesome, thanks!

File: src/webgl/p5.Camera.js [hunk 1]
```diff
@@ -345,9 +345,22 @@ p5.prototype.frustum = function(...args) {
...
+ *   background(0);
+ *   // The camera will automatically
+ *   // rotate to look at [0, 0, 0].
  *   camera.lookAt(0, 0, 0);
+ * 
+ *   // The camera will move on the
+ *   // x axis.
  *   camera.setPosition(sin(frameCount / 60) * 200, 0, 100);
  *   box(20);
+ *
```
Comment (davepagurek) [inline src/webgl/p5.Camera.js:357]: oh sorry, one more thing, I think tests are failing because there are trailing spaces on this line

Comment (davepagurek) [reply src/webgl/p5.Camera.js:357]: and also line 352, I think? If you run `npm run lint` I believe it'll let you know when you've got all of them!

Comment (Brahvim) [reply src/webgl/p5.Camera.js:357]: Hmm... lines `352` and `357`. I removed the trailing spaces, let's see if this gets through!

**Comment by Brahvim** [comment]
I accidentally ran a code format on my local copy, and "[Linting fix number 2.](https://github.com/processing/p5.js/pull/5873/commits/2ea2bef881b3cda196398e5809dd57a9a7c069a1)" should fix that. I copy-pasted a previous version of the file and re-did the whitespace removal. Let's see how this goes...

**Comment by davepagurek** [comment]
ah hmm it looks like the most recent change indented everything by a space so now the whole file appears in the diff. It might be that the easiest fix  might be to revert the last two commits and try removing the spaces again? sorry for all the formatting back-and-forth, I can also try getting the formatting back to what it needs to be if that's easier!

**Comment by Brahvim** [comment]
I apologize! I tried doing a similar thing earlier (copy-pasting a previous version, making changes and uploading that, as seen in "[Linting fix number 2.](https://github.com/processing/p5.js/pull/5873/commits/2ea2bef881b3cda196398e5809dd57a9a7c069a1)"). Clearly, that did not work.

Your help would very-much be appreciated.

**Comment by davepagurek** [comment]
No problem 🙂

**Comment by davepagurek** [comment]
@all-contributors please add @Brahvim for doc

**Comment by Brahvim** [comment]
Gratitudes from my side - thank you a lot, @davepagurek!

**Pull request merged**